### ISSUE TITLE
Incorrect formatting for @_documentation

### ISSUE THREAD

**Issue by rauhul** [apple/swift-format#735]
swift-format re-formats the following attribute:
```swift
@_documentation(visibility:       internal)
```
as:
```swift
@_documentation(visibility:internal)
```
Instead of:
```swift
@_documentation(visibility: internal)
```

**Comment by allevato** [comment]
This was fixed by https://github.com/apple/swift-format/pull/719, so it should be available in the `release/6.0` branch.

**Comment by rauhul** [comment]
Sorry for the noise!
